source/triggers/aws-eventbridge.txt

```diff
@@ -152,6 +168,233 @@ Procedure
+                  }
+
+                  console.log('Error sending event to EventBridge');
+                  console.log(`DB: ${changeEvent.ns.db}`);
+                  console.log(`Collection: ${changeEvent.ns.coll}`);
+                  console.log(`Operation type: ${changeEvent.operationType}`);
+
+                  // Throw an error in your function to suspend the trigger and stop processing additional events
+                  throw new Error(`Encountered error: ${error.message}`);
+               };
+
+         .. step:: Add an Error Handler to Your Trigger Configuration
+
+            Add an ``error_handler`` attribute to your trigger configuration file
+            in the ``Triggers`` folder. The trigger configuration file should 
+            resemble the following:
+
+            .. code-block:: json
+               :emphasize-lines: 13-18
+
+                     {
+                     "name": "...",
+                     "type": "...",
+                     "event_processors": {
+                        "AWS_EVENTBRIDGE": {
+                              "config": {
+                                 "account_id": "<AWS Account ID>",
+                                 "region": "<AWS Region>",
+                                 "extended_json_enabled": <boolean>
+                              }
+                           }
+                        },
+                        "error_handler": {
+                           "config": {
+                              "enabled": <boolean>,
+                              "function_name": "<Error Handler Function Name>"
+                           }
+                        }
+                     }
+            
+            For more information on trigger configuration files, see
+            :ref:`Trigger Configuration Files <appconfig-triggers>`.
+
+.. _eventbridge-error-handler-parameters:
+
+Error Handler Parameters
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The default error handler has two parameters: ``error`` and ``changeEvent``.
+
+``error`` 
+`````````
+Has two attributes: ``code`` and ``message``.
+
+- ``code``: For a list of error codes used by the error handler, see the below section on
+  :ref:`Error Codes <eventbridge-error-codes>`. 
+
+- ``message``: The unfiltered error message from an errored Eventbridge
+  put request.
+
+``changeEvent``
+```````````````
+
+The requested change to your data made by EventBridge. For more information
+on types of change events and their configurations, see 
+:ref:`Change Event Types <database-events>`.
+
+.. _eventbridge-error-codes:
+
+Error Codes
+~~~~~~~~~~~
+
+If an error was recevied from EventBridge, the event processor will parse the 
+error as either ``DOCUMENT_TOO_LARGE`` or ``OTHER``. This parsed error is passed
+to the error handler function through the ``error`` parameter. 
+
+``DOCUMENT_TOO_LARGE``
+``````````````````````
+If the put entry for an EventBridge trigger event is larger
```

Comment:
do we want to link down to the Performance Optimization section where we tell them how to reduce doc size?